test/WorkItemBuiltins/multiple-push-constant.cl

```diff
@@ -0,0 +1,34 @@
+// RUN: clspv -work-dim -global-offset -descriptormap=%t.dmap %s -o %t.spv
```

Comment:
Another variant of this test please, call get_global_offset with a larger index (or non-constant index) to generate the select.

Reply:
I've ended up doing another variant of the simple `get_global_offset` test with a non-constant index. I didn't feel having get_work_dim in the same test added anything.